webpack/assets/javascripts/react_app/components/BreadcrumbBar/BreadcrumbBarActions.js

```diff
@@ -83,6 +83,7 @@ export const loadSwitcherResourcesByResource = (
       per_page: 10,
       search: createSearch(nameField, searchQuery, resource.resourceFilter),
     }
+    // eslint-disable-next-line promise/prefer-await-to-then
```

Comment:
I'm concerned that if we add these everywhere we'll never actually make this change.  Would it be that much more work to convert each occurrence?

Reply:
I prefer to add this rule now (with an example) so a new code will be formatted by this.

Reply:
@waldenraines thought?

Reply:
Isn't it possible to fix them first and then enable the rule? Personally I'm a fan of limiting the noise in git log.

Reply:
@ekohl  - enabling the rule now prevents new code from having `then` syntax for promises. I prefer to progress on the UI road map, instead of fixing eslint rules (especially when it's not a trivial thing), at least we will enforce new code from using the old syntax, so the migration to `async - await` would be much easier.

Reply:
> Isn't it possible to fix them first and then enable the rule? Personally I'm a fan of limiting the noise in git log.

Yes, but there is a trade-off in time spent fixing the existing occurrences.  To me, adding a rule to prevent future occurrences and providing an example so that others can refactor when they are working in the existing code is a fine way forward.  

Basically it comes down to these options:

- Adding this now and fixing these occurrences as we go
- Allowing our technical debt to increase by not adding it now and hope that we have time to fix all of the occurrences in the future

To me the choice is clear and having one extra commit in the log is not a good reason to prevent this from going in.

Reply:
I don't feel strongly about this. Ideally we'd avoid the ignore rules but I can see why these lint rules aren't trivial replacements and a good chance to introduce bugs (unlike visual things like trailing commas).